catalog/core/catalog-core-api/src/main/java/ddf/catalog/CatalogFramework.java

```diff
@@ -41,7 +43,6 @@
 import ddf.catalog.source.SourceUnavailableException;
 import ddf.catalog.source.UnsupportedQueryException;
 import ddf.catalog.transform.CatalogTransformerException;
-import ddf.catalog.util.Describable;
```

Comment:
We can't do this. This will be a breaking change to anyone else that has implemented CatalogFramework.